compiler/luci-value-test/test.lst

```diff
@@ -54,8 +54,8 @@ addeval(Exp_000)
 addeval(Floor_000)
 addeval(FloorDiv_000)
 addeval(FloorDiv_001)
-#addeval(FloorMod_000)
-#addeval(FloorMod_001)
+addeval(FloorMod_000)
+addeval(FloorMod_001)
```

Comment:
plz split this to another PR.
`compiler` has policy not to mix changes of `compiler/*` in one PR.

Reply:
OK. I will.